.github/workflows/tests.yml

```diff
@@ -60,7 +68,7 @@ jobs:
         BASE_DOMAIN: ${{ secrets.DELTA_BASE_DOMAIN }}
       run: |
         export NAME=cdk-deploy-${GITHUB_SHA:0:6}
-        ./util.py generate_config_template --name $NAME --aws-region=us-west-2 --aws-account-id=$AWS_ACCOUNT_ID --dev --registry-username $REGISTRY_USERNAME --registry-password $REGISTRY_PASSWORD --hostname $NAME.$BASE_DOMAIN --acm-cert-arn $ACM_CERT_ARN --disable-flow-logs > config.yaml
+        ./util.py generate_config_template --name $NAME --aws-region=us-west-2 --aws-account-id=$AWS_ACCOUNT_ID --dev --platform-nodegroups 2 --registry-username $REGISTRY_USERNAME --registry-password $REGISTRY_PASSWORD --hostname $NAME.$BASE_DOMAIN --acm-cert-arn $ACM_CERT_ARN --disable-flow-logs > config.yaml
```

Comment:
Why 2 nodegroups?

Reply:
Just simplest way to get us two default nodes without either doing some template override/sed shenanigans or adding more args to the template generator, and I'm only doing this as a temporary thing so I didn't want it to be the need that drives a `--platform-min` type arg.